### PR TITLE
Update header partial

### PR DESCRIPTION
**Pull request by jacquesporveau**
I've added a sign in/out link to the header

Pivotal: https://www.pivotaltracker.com/story/show/142092931

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -4,6 +4,13 @@
     <ul>
       <li><%= link_to 'Upcoming Events', upcoming_events_path %></li>
       <li><%= link_to 'New Presentation', new_presentation_path %></li>
+      <li>
+        <% if current_person %>
+          <%= link_to 'Sign Out', signout_path %>
+        <% else %>
+          <%= link_to 'Sign In', '/auth/google_oauth2' %>
```
Comment (graygilmore) [inline app/views/layouts/_header.html.erb]: Is there a defined path for this in our routes?

Comment (jacquesporveau) [inline app/views/layouts/_header.html.erb]: Yep it's `get 'auth/:provider/callback', to: 'authentication#google'`

Comment (graygilmore) [reply app/views/layouts/_header.html.erb]: Let's use the `_path` helper then. A little better than manually specifying the string.

Comment (jacquesporveau) [reply app/views/layouts/_header.html.erb]: I think the gem adds that route in for us or something. It's not defined in our routes but I've tested it and it works / this is old behaviour from the google oauth PR being re introduced.

**Comment by jacquesporveau** [comment]
For clarities sake the path that I've overwritten in the routes file was not being user anywhere.

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (jacquesporveau) [reply app/views/layouts/_header.html.erb]: Sounds good I've updated the PR to include an aliased route.

**Review by qr8r — APPROVED** [review]

**Pull request merged**